src/core/jira-manager.ts

```diff
@@ -134,12 +123,13 @@ export class JiraManager {
           const tickets: JiraTicket[] = data.issues.map((issue) => ({
             id: issue.id,
             key: issue.key,
-            summary: issue.fields.summary,
+            summary: issue.fields.summary || '',
             description: this.extractDescriptionText(issue.fields.description),
-            status: issue.fields.status.name,
+            status: issue.fields.status?.name || '',
             assignee: issue.fields.assignee?.displayName,
-            created: issue.fields.created,
-            updated: issue.fields.updated,
+            created: issue.fields.created || '',
+            updated: issue.fields.updated || '',
```

Comment:
duckling these manually added ones like summary, status, created and updated etc aren't used directly anywhere. we don't need to store it since it'll be in all fields anyway